test/functional/apps/visualize/_histogram_request_start.js

```diff
@@ -1,95 +0,0 @@
-/*
- * SPDX-License-Identifier: Apache-2.0
- *
- * The OpenSearch Contributors require contributions made to
- * this file be licensed under the Apache-2.0 license or a
- * compatible open source license.
- *
- * Any modifications Copyright OpenSearch Contributors. See
- * GitHub history for details.
- */
-
-/*
- * Licensed to Elasticsearch B.V. under one or more contributor
- * license agreements. See the NOTICE file distributed with
- * this work for additional information regarding copyright
- * ownership. Elasticsearch B.V. licenses this file to you under
- * the Apache License, Version 2.0 (the "License"); you may
- * not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *    http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing,
- * software distributed under the License is distributed on an
- * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
- * KIND, either express or implied.  See the License for the
- * specific language governing permissions and limitations
- * under the License.
- */
-
-import expect from '@osd/expect';
-
-export default function ({ getService, getPageObjects }) {
-  const log = getService('log');
-  const retry = getService('retry');
-  const PageObjects = getPageObjects([
-    'common',
-    'visualize',
-    'visEditor',
-    'visChart',
-    'timePicker',
-  ]);
-
-  describe('histogram agg onSearchRequestStart', function () {
```

Comment:
were these tests also migrated to the functional test repo, even though they're not tests for the table vis itself?

Reply:
It hasn't yet. But I am working on it. 
Currently, all the table functions and tests on visualization page are re-produced. Things left: 
1) embedding a table ([_embedding_chart.js](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/2863/files/73a636a97413218b2f1a00a784b26dc59210fc81#diff-288a4d30a60dccd027c76326c82db67ab25e9ed0df154c76ca36c8559674031d))
2) some aggregation tests via table vis if there is no better way to test ([_histogram_request_start.js](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/2863/files/73a636a97413218b2f1a00a784b26dc59210fc81#diff-9d8e3b4473cbdfef31b5dab938185016b18bd21b50e6d7ca3e19301e2b718b11))
3)add table to dashboard page
4)saved search from discover